extension/intents/tabs/tabs.toml

```diff
@@ -454,3 +450,36 @@ expectParameters = { dir = "first" }
 [[tabs.selectNumbersTabs.example]]
 phrase = "select last two tabs"
 expectParameters = { dir = "last" }
+
+[tabs.closeAll]
+description = "Closes all active or open tabs"
+match = """
+  close (all | every | each | the |) (active | open |) (tab{s} | site{s} | page{s}) (for me |)
```

Comment:
This should only match "close all tabs" or maybe "close every tab". So something like `close (all | every) tab{s} (for me|)`

Reply:
done!